src/renderer/base/FontInfoBase.cpp

```diff
@@ -100,28 +96,32 @@ void FontInfoBase::ValidateFont()
     if (!IsDefaultRasterFontNoSize() && s_pFontDefaultList != nullptr)
     {
         // If we have a list of default fonts and our current font is the placeholder for the defaults, substitute here.
-        if (0 == wcsncmp(_wszFaceName, DEFAULT_TT_FONT_FACENAME, ARRAYSIZE(DEFAULT_TT_FONT_FACENAME)))
+        if (_faceName == DEFAULT_TT_FONT_FACENAME)
         {
-            WCHAR pwszDefaultFontFace[LF_FACESIZE];
+            std::wstring defaultFontFace;
             if (SUCCEEDED(s_pFontDefaultList->RetrieveDefaultFontNameForCodepage(GetCodePage(),
-                                                                                 pwszDefaultFontFace,
-                                                                                 ARRAYSIZE(pwszDefaultFontFace))))
+                                                                                 defaultFontFace)))
             {
-                wcscpy_s(_wszFaceName, ARRAYSIZE(_wszFaceName), pwszDefaultFontFace);
+                _faceName = defaultFontFace;
 
                 // If we're assigning a default true type font name, make sure the family is also set to TrueType
                 // to help GDI select the appropriate font when we actually create it.
-                _bFamily = TMPF_TRUETYPE;
+                _family = TMPF_TRUETYPE;
             }
         }
     }
 }
 
 bool FontInfoBase::IsTrueTypeFont() const
 {
-    return (_bFamily & TMPF_TRUETYPE) != 0;
+    return WI_IsFlagSet(_family, TMPF_TRUETYPE);
 }
 
+#pragma warning(push)
+#pragma warning(suppress : 4356)

```

Comment:
Do we plan to permanently suppress this (and leave justification in comment)? Or should there be a todo on it? (and leave that in comment)

Reply:
Looked into this warning: it actually isn't necessary to suppress for this use, so we can demolish it. I migrated it from the original code.